Controller/CRUDController.php

```diff
@@ -208,17 +240,19 @@ protected function handleModelManagerException(\Exception $e)
     /**
      * List action
      *
+     * @param Request $request
+     *
      * @return Response
      *
      * @throws AccessDeniedException If access is not granted
      */
-    public function listAction()
+    public function listAction(Request $request)
```

Comment:
Can you make the request optional ? This will be a big BC break for the code extending CRUD method.